{"name": "Проверка: если передать несуществующий id, вернётся ошибка 404", "status": "passed", "start": 1710697539748, "stop": 1710697540241, "uuid": "b81971fd-8dd9-4685-973a-15ae6a9c1047", "historyId": "0b194bc681249f4a894c19eaac46cb90", "testCaseId": "0b194bc681249f4a894c19eaac46cb90", "fullName": "tests.tests_courier.test_delete_courier.TestDeleteCourier#test_delete_courier_with_invalid_id_causes_404_error", "labels": [{"name": "parentSuite", "value": "tests.tests_courier"}, {"name": "suite", "value": "test_delete_courier"}, {"name": "subSuite", "value": "TestDeleteCourier"}, {"name": "host", "value": "Olyas-MacBook-Air.local"}, {"name": "thread", "value": "94503-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_courier.test_delete_courier"}]}